test/Interop/Cxx/concepts/method-requires.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %target-swiftxx-frontend -emit-ir -Xcc -std=gnu++20 -I %S/Inputs %s | %FileCheck %s
+//
+// REQUIRES: OS=macosx
```

Comment:
Can you add a note saying why this needs macOS in C++ 20 mode . Thanks!

Reply:
Filed https://github.com/apple/swift/issues/65710